spec/javascript/components/organisms/StatusModalWrapper.test.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
+import StatusModalWrapper from '~/components/organisms/StatusModalWrapper'
+import FormModal from '~/components/organisms/FormModal'
+import { LEASE_UP_SUBSTATUS_OPTIONS } from '~/utils/statusUtils'
+
+const statusUtils = require('~/utils/statusUtils')
```

Comment:
I assume we need to import with `require` because of jest? Can you make a comment explaining that?